sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/jni/BranchIdTest.kt

```diff
@@ -49,16 +49,16 @@ class BranchIdTest internal constructor(
             val mainnetBackend = runBlocking { RustBackend.init("", "", "", ZcashNetwork.Mainnet, ZcashNetwork.Mainnet.saplingActivationHeight) }
             return listOf(
                 // Mainnet Cases
-                arrayOf("Sapling", 419_200, 1991772603L, "76b809bb", mainnetBackend),
-                arrayOf("Blossom", 653_600, 733220448L, "2bb40e60", mainnetBackend),
-                arrayOf("Heartwood", 903_000, 4122551051L, "f5b9230b", mainnetBackend),
-                arrayOf("Canopy", 1_046_400, 3925833126L, "e9ff75a6", mainnetBackend),
+                arrayOf("Sapling", BlockHeight(419_200), 1991772603L, "76b809bb", mainnetBackend),
```

Comment:
Ideally, we'd use `BlockHeight.new(ZcashNetwork.Mainnet, X)`

The internal constructor was primarily for a very specific usage inside the SDK, and the `new` function is safer.